### DISCUSSION TITLE
How can I configure dataset to train BEV from scratch

### DISCUSSION THREAD

**Discussion by 170928** [Arthur151/ROMP#327] [Q&A] [answered]
Hi, @Arthur151! 
I really appreciate you to share this wonderful research implementation.

Recently, I carefully try to understand your BEV implementation. Thanks to you, I can understand some complex points :) (in Issue #322  #321 ) 

And, I have some questions about dataset to train BEV from scratch. 
In your paper, a total of 7 datasets are used to train BEV.
1. two 3D dataset : 
   (1) Human3.6m 
   (2) Muco-3DHP
2. four 2D dataset : 
   (1) COCO
   (2) MPII
   (3) LSP
   (4) CrowdPose 
3. fine-tune using AGORA 

I think Human3.6m, Muco-3DHP has depth information; thus I can use these datasets for 
(0) L_{cm} : loss using front-view center map
(1) L_{cm3D} : loss using bird's eye view center map
(2) L_{j3d} : loss using 3D joints 
(3) L_{pj2d} : loss using the 2D projection of 3D joints 
(4) L_{pm} : loss consists of L_\theta, L_\beta, L_\prior 

I have questions at here.
(1)  I think only AGORA can use L_{pm} because other datasets don't have SMPL ground truth information. 
       Thus, two 3D dataset use (0) ~ (3) loss and four 2D dataset use (0) and (3) loss.
       After that, in finetuning stage, we use AGORA dataset and (4) loss for SMPL parameter inference, is it right?.

(2) How did you make SMPL ground truth if other data also had SMPL ground truth?

I want to carefully understand your paper and implementation. After that, I wanna configure simple training code and contribute your project, if you don't mind :)

**Comment by Arthur151** [comment, marked as answer]
Thanks for your interesting in BEV! Really enjoy the valuable discussions with you. 

About the training dataset, Relative human datasets provide some weak annotations of some images from 2D pose datasets, including COCO, CrowdPose, to learn multi-person relative depth relationships. 
Besides, thanks for previous researchers, we have the SMPL annotations of Human3.6M (provided by SOMA) and pseudo SMPL annotations of some images of 2D pose datasets (provided by EFT). So we can supervise the theta (SMPL pose parameters).

Welcome to contribute to this rep. 
Please note that some complex operation is not just developed for ROMP / BEV. Some complex function is developed for the next work I am working on.
